osquery/tables/system/freebsd/mounts.cpp

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Instead of copy-pasting the darwin implementing: `ln -s ../darwin/mounts.cpp .`. Same as: https://github.com/facebook/osquery/blob/master/osquery/tables/system/freebsd/groups.cpp